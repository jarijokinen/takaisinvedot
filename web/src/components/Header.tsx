import * as Styled from './Header.styled';

const Header: React.FC = () => (
  <Styled.Header>
    <Styled.Brand>Takaisinvedot.fi</Styled.Brand>
    <Styled.Slogan>
      Viallisten tuotteiden ja elintarvikkeiden takaisinvedot koottuna yhdelle
      sivulle
    </Styled.Slogan>
  </Styled.Header>
);

export default Header;
