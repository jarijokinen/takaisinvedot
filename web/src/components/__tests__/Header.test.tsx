import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import Header from '../Header';

describe('Header', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
