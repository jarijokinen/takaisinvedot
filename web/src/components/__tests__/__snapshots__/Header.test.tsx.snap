// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<DocumentFragment>
  <header
    class="sc-AxjAm iLAjRt"
  >
    <h1
      class="sc-AxirZ cniLYN"
    >
      Takaisinvedot.fi
    </h1>
    <div
      class="sc-AxiKw lhOBPC"
    >
      Viallisten tuotteiden ja elintarvikkeiden takaisinvedot koottuna yhdelle sivulle
    </div>
  </header>
</DocumentFragment>
`;
